x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -135,7 +136,7 @@ IS: 'is';
 LAST : 'last';
 LIKE: 'like';
 LP : '(';
-MATCH: 'match';
+MATCH_OPERATOR: 'match';
```

Comment:
I changed this so it doesn't collide with the `MATCH` command - even though it's the same lexer definition, I'm unsure about the `pushMode(EXPRESSION_MODE)` that is done in the command definition. I thought it safer to differentiate the two, but LMK if I shouldn't

Reply:
If we could use a different name for the `MATCH` command, that will reduce confusions, as there is already a `MATCH` operator, and they don't do exactly the same thing. It could be addressed as a follow up issue.

Reply:
Agreed @fang-xing-esql . Naming is hard 😓

Reply:
That's interesting! Why does that happen, is it something missing from the lexer definition?